ulog/doc.go

```diff
@@ -21,7 +21,7 @@
 // Package ulog is the Logging package.
 //
 // package ulog provides an API wrapper around the logging library
-// zap (https://github.com/uber-go/zap). package ulog uses the builder pattern
+// zap (https://go.uber.org/zap). package ulog uses the builder pattern
```

Comment:
Also, these files are auto generated (`make gendoc`) from README so if you do want to change it, change it there first